### ISSUE TITLE
[FeatureRequest] Add support for M309 Heater Feedforward to expansion and tool boards

### ISSUE THREAD

**Issue by droftarts** [dc42/RepRapFirmware#1030]
Heater feedforward is currently only supported on heaters connected to mainboard:
```
void SetExtrusionFeedForward(float pwm) noexcept override { }            // We can't yet set feedforward on remote heaters because this is called from an ISR
```

Suggestion from user (on Discord) to add support for remote heaters. Also reported on forum https://forum.duet3d.com/topic/36180/implement-m309-feedforward-for-remote-heaters (same user).

@dc42 said
> It might not be difficult to implement where the extruder and heater are driven from the same board e.g a tool board.

**Comment by dc42** [comment]
Implemented yesterday in 3.6-dev, not tested yet.

**Comment by dc42** [comment]
Tested and working.
